doc/lua_api.md

```diff
@@ -93,6 +93,10 @@ The game directory can contain the following files:
                  an internal ID used to track versions.
     * `textdomain`: Textdomain used to translate description. Defaults to game id.
       See [Translating content meta](#translating-content-meta).
+	* `aliases = <comma-separated aliases>`
+	  e.g. `aliases = aa,bb` (game has been named "aa" and then "bb" before)
```

Comment:
this makes it sound like the order is relevant. is it?

Reply:
It should also clarify that it's the technical game name, `gameid`. For example:
```
	* `aliases = <comma-separated gameid aliases>`
	  e.g. `aliases = foo, bar` (where "foo" and "bar" are the legacy names)
```